### PR TITLE
Apple page

### PR DESCRIPTION
**Pull request by yahir91**
This project is a clone of an old landing page of apple. This page structure is built using CSS grid and flexbox and I had to use linear-gradient to match the navbar as the original.

### CODE REVIEW

**Comment by rammazzoti2000** [comment]
## Pull Request under review byAlexandru Bangau (TSE)

File: style.css [hunk 1]
```diff
@@ -0,0 +1,220 @@
+main {
+  display: grid;
+  gap: 2px;
+  grid-template-columns: repeat(4 1fr);
```
Comment (rammazzoti2000) [inline style.css]: missed the comma after 4
`grid-template-columns: repeat(4, 1fr)`

File: style.css [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+  grid-template-areas:
+    "main main main main"
+    "one two three four";
+  width: 1440px;
```
Comment (rammazzoti2000) [inline style.css]: this width is causing heavy horizontal overflow, remove it to fix the issue

File: style.css [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+nav a,
+nav div {
+  display: inline-block;
+  flex-grow: 1;
+  box-shadow: 0.5px 0 0.1px 0.04px #949494;
+  vertical-align: middle;
+  position: relative;
+}
+
+nav a {
```
Comment (rammazzoti2000) [inline style.css:79]: remove text-decoration for elements in navbar
`text-decoration: none;`

File: style.css [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+  line-height: 18px;
+  font-weight: 100;
+}
+
+p img {
+  height: 25px;
+}
+
+footer {
+  width: 1000px;
```
Comment (rammazzoti2000) [inline style.css]: change width to 
```
width: -webkit-fit-content;
width: -moz-fit-content;
width: fit-content;
```

will be cross platform(browser) compatible and will use the space available but never less than the min-content and never more than the max-content"

File: style.css [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+  margin: 10px auto 0;
+}
+
+.second-footer {
+  display: inline-block;
+  margin-left: 90px;
+  font-weight: 400;
```
Comment (rammazzoti2000) [inline style.css]: this it seems it has no effect since 400 is the same as normal thickness, you can remove it

File: style.css [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+  font-size: 10px;
+  font-family:
+    "Lucida Grande",
+    "Lucida Sans Unicode",
+    Helvetica,
+    Arial,
+    Verdana,
+    sans-serif;
+  line-height: 12px;
+  font-weight: 400;
```
Comment (rammazzoti2000) [inline style.css]: this it seems it has no effect since 400 is the same as normal thickness, you can remove it

File: style.css [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+}
+
+footer a {
+  color: #08c;
+  font-size: 10px;
+  line-height: 12px;
+  font-weight: 400;
```
Comment (rammazzoti2000) [inline style.css]: this it seems it has no effect since 400 is the same as normal thickness, you can remove it

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### Hello @yahir91 :wave: my name is Alexandru, your code reviewer for this project
Your project is really great, you used HTML semantic, no css bad practices (repeating rules) except for one only, great job on the use of css background and gradients :+1: 
The project is almost ready for approval except for some small issues :eyes: 
### Required Changes :construction: :construction_worker: 
- [ ] The page has an horizontal overflow, remove it by removing the width in the main selector
- [ ] Remove text-decoration for the elements in navbar `text-decoration: none;`
- [ ] The width for the footer is better to change it to `fit-content` so that it uses only the space needed based on the available space, see the inline-comment bellow :point_down: 
- [ ] Also you have a repeating rule (`font-weight: 400`) in the footer anyway the value 400 refers to the normal font-weight so you don't need it, remove it; see the inline comments bellow :point_down: 
### Optional :smile: 
- [ ] Would be great to have a live demo link in the readme file
### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

##### Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>

### :stop_sign:  Status: Changes required.

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### :stop_sign: Status: Required Changes

**Review by trekab — APPROVED** [review]
# Approved! :tada: 
Congratulations, @team!!! :clap: You may merge to master, submit the project completion form and continue to the next project.

Happy coding! :wink:
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Pull request merged**